corporate/calendar.rst

```diff
@@ -1,27 +1,90 @@
+:nosearch:
+:og:description: This page describes tasks that must be completed on a set schedule.
+
 Calendar of tasks
 =================
 
-.. todo:: finish writing this page
-
+.. vale Google.LyHyphens = NO
 .. vale Google.Passive = NO
-.. vale write-good.Passive = NO
 .. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+
+This page describes tasks that must be completed on a set schedule.
+
+January
+-------
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+* Submit Sales & Use Tax Return to the Georgia Department of Revenue
+* `File Annual Registration with the Georgia Secretary of State <https://sos.ga.gov/how-to-guide/how-file-annual-registration>`_
+* Hold quarterly board meeting
+* Reconcile 1099-K received from Square to QuickBooks records
+
+February
+--------
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+
+March
+-----
+
+.. vale Google.Units = NO
+.. vale Google.Ordinal = NO
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+* `Annual Registration with the Georgia Secretary of State <https://sos.ga.gov/how-to-guide/how-file-annual-registration>`_ due by April 1st
+
+April
+-----
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+* Hold quarterly board meeting
+
+May
+---
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+* Renew Northwest Registered Agent contract
+
+June
+----
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+
+July
+----
+
+.. vale write-good.TooWordy = NO
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+* Begin compiling documentation for Internal Revenue Service Form 990-EZ
+* Hold quarterly board meeting - elect directors and officers
+
+August
+------
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+* Audit BILL Spend & Expense cardholders
```

Comment:
Mention collaboration with student officers?

Reply:
Added link to stub page